src/IlkRegistry.sol

```diff
@@ -107,17 +110,18 @@ contract IlkRegistry {
     // Pass a dss End contract to the registry to initialize
     constructor(address end) public {
```

Comment:
I feel from a semantic point of view `end` shouldn't be use as the source, thinking that we do not even keep it saved. I'd prefer the constructor would be the `vat`, `cat` and `spot` and make the necessary checks to see everything matches. That would feel much more explicit. Anyway this can be discussed later,

Reply:
We can do that, I just went with the `end` for brevity since it has everything we need and we can make some assumptions that they are all tied together.